tools/jil/util/external-services.js

```diff
@@ -28,60 +25,25 @@ function startExternalServices (browsers, config, cb) {
 
   if (!needSauce) return cb()
 
-  let remaining = [needSauce].filter(Boolean).length
-
-  if (needSauce) startSauce(config, checkDone)
-
-  function checkDone () {
-    if (--remaining) return
-    cb()
-  }
+  import('../../saucelabs/utils.mjs')
```

Comment:
Asking for a friend, can you mix `require()` and `import()` in the same file like this?

Reply:
You have to use `import()` since this is a CJS file and its pulling in an ESM file.